app/Http/Controllers/CampaignsController.php

```diff
@@ -58,6 +58,11 @@ public function index(Request $request)
             }
         }
 
+        // Apply scope for campaigns including specified cause:
+        if (isset($filters['has_cause'])) {
```

Comment:
Sorry for not thinking about this earlier, but would it be clearer to have this as just `cause`?